### PR TITLE
DevKit updates for 2.x branch

### PR DESCRIPTION
**Pull request by SonataCI**

### CODE REVIEW

File: tests/Helper/DateTimeHelperTest.php [hunk 1]
```diff
@@ -31,7 +31,7 @@ public function testLocale()
     {
         $localeDetector = $this->createMock('Sonata\IntlBundle\Locale\LocaleDetectorInterface');
         $localeDetector->expects($this->any())
-            ->method('getLocale')->will($this->returnValue('fr'));
```
Comment (greg0ire) [inline tests/Helper/DateTimeHelperTest.php]: What was wrong with french?

Comment (jordisala1991) [reply tests/Helper/DateTimeHelperTest.php]: It produces a lot of errors with whitespaces for some reason

Comment (jordisala1991) [reply tests/Helper/DateTimeHelperTest.php]: normal char for whitespace: char(32). French char for whitespace char(194) char(160) (or something)

cs-fixer was replacing all whitespaces for the normal value, then the test were failing

Not sure how else fix this @greg0ire

Comment (jordisala1991) [reply tests/Helper/DateTimeHelperTest.php]: captura de pantalla 2017-12-07 a las 22 41 05

See how cs-fixer is replacing whitespaces with another whitespace?

**Pull request merged**